scripts/rook.sh

```diff
@@ -204,6 +222,12 @@ create-block-pool)
 delete-block-pool)
 	delete_block_pool
 	;;
+create_ec_pool)
```

Comment:
where does this get called?

Reply:
this is not called anywhere. @Yuggupta27 as you are already creating the ec-pool by default when deploying rook.

you need to remove the creating by default because the ec-pool works only for 3 osd. add this new pool creation to ci scripts in centos branch?

Reply:
This just enhances the `rook.sh` scripts functionality to allow the creation of erasure-coded pool using the script; similar to what we have for [creating block pools.](https://github.com/ceph/ceph-csi/blob/devel/scripts/rook.sh#L201-L206)

Reply:
Nice Catch! Will send a PR shortly for deploying only in CI

Reply:
Why do tests pass if the ec-pool is not created?